service-worker.js

```
@@ -9,7 +9,7 @@ const operationResolver = new OperationResolver(storage);
 chrome.runtime.onMessage.addListener(async function (message, sender, sendResponse) {
   switch (message.type) {
     case 'gestures':
-      const context = new Context(message.gestures, message.selectedText);
+      const context = new Context(message.gestures, message.selectedText, message.linkUrl);
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)